Source/WebCore/platform/sql/SQLiteTransaction.cpp

```diff
@@ -27,10 +27,7 @@
 #include "SQLiteTransaction.h"
 
 #include "SQLiteDatabase.h"
-
-#if PLATFORM(IOS_FAMILY)
```

Comment:
Should this be `#if PLATFORM(COCOA)`? Or does this stuff exist/work on GTK/Windows?

Reply:
Looking at the tracker code, it seems cross-platform, so this seems right.

Reply:
The SQLiteDatabaseTracker builds fine on all platforms but is a no-op on some platforms. It doesn't hurt to run this code everywhere and dropping the `#if` checks makes the code nicer.